public/js/reader.js

```diff
@@ -308,6 +338,9 @@ Reader.updateMetadata = function () {
     const img = $("#img")[0];
     const imageUrl = new URL(img.src);
     const filename = imageUrl.searchParams.get("path");
+    const img_doublepage = $("#img_doublepage")[0];
```

Comment:
I forgot to mention this in the previous review, but I try to stick to camelCase for variable names on the JS side of things.  

It'd be awesome if you could update the code to be as compliant as possible with the rules in https://github.com/Difegue/LANraragi/blob/dev/.eslintrc.json ! 🙏